src/shared/components/palette/utils/is-read-only-and-not-collection.pipe.ts

```diff
@@ -19,6 +19,7 @@ export class IsReadOnlyAndNotCollectionPipe implements PipeTransform {
     return this.caseFieldService.isReadOnly(field);
   }
 
+  // CaseField @Expose() doesn't work with the pipe in here, so leaving the manual check
```

Comment:
Why this comment here?

Reply:
if (field.isCollection()) { has been failing in the console with an error. We use class-transformer to expose isCollection() in CaseField model, but in the above pipe it doesn't work. The comment is there to make a person think before changing it.